tracer/build/_build/docker/gitlab/entrypoint.bat

```diff
@@ -0,0 +1,42 @@
+@echo off
```

Comment:
Combines the existing `entrypoint.bat` script with our previous gitlab.bat file, but no changes to the contents really